Library/Formula/hashpump.rb

```diff
@@ -10,13 +10,23 @@ class Hashpump < Formula
     sha256 "cf4a9cee9d04bef03016073dc0e516e5349861691ad80bae1706cbd846b03920" => :mountain_lion
   end
 
+  option "with-python", "Build with Python bindings"
+
   depends_on "openssl"
+  depends_on :python => :optional
 
   def install
     bin.mkpath
     system "make", "INSTALLLOCATION=#{bin}",
                    "CXX=#{ENV.cxx}",
                    "install"
+
+    if build.with? "python"
+      # The Python bindings require the headers, which are named generically and
+      # can conflict with other package's headers.
```

Comment:
`system "python", *Language::Python.setup_install_args(libexec)`.

[This](https://github.com/bwall/HashPump/blob/master/setup.py) is the upstream `setup.py` file.